.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           name: Run the BigQuery Indexer
           command: |
             echo ${GOOGLE_SERVICE_KEY} | base64 --decode > ${HOME}/.config/gcloud/application_default_credentials.json
-            docker network create dataexplorer_default
+            docker network create data-explorer_default
```

Comment:
Why not data_explorer_default or data-explorer-default for consistency?

Reply:
docker-compose automatically generates this name. It takes directory name and adds "_default".